flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/common/CommonExecSink.java

```diff
@@ -161,28 +167,50 @@ public DynamicTableSinkSpec getTableSinkSpec() {
     /**
      * Apply an operator to filter or report error to process not-null values for not-null fields.
      */
-    private Transformation<RowData> applyNotNullEnforcer(
+    private Transformation<RowData> applyConstraintValidations(
             Transformation<RowData> inputTransform, TableConfig config, RowType physicalRowType) {
-        final ExecutionConfigOptions.NotNullEnforcer notNullEnforcer =
-                config.getConfiguration()
-                        .get(ExecutionConfigOptions.TABLE_EXEC_SINK_NOT_NULL_ENFORCER);
-        final int[] notNullFieldIndices = getNotNullFieldIndices(physicalRowType);
+        final ConstraintValidator.Builder validatorBuilder = ConstraintValidator.newBuilder();
         final String[] fieldNames = physicalRowType.getFieldNames().toArray(new String[0]);
+        String operatorName = "";
 
+        final int[] notNullFieldIndices = getNotNullFieldIndices(physicalRowType);
```

Comment:
Add a comment above to specify you're going to buidl the not null enforcer here, and same below to say you're going to build the string length enforcer